pipes/fetch-job-scripts.pipe.js

```diff
@@ -91,7 +91,7 @@ const getJobScripts = async (context, job) => {
       scriptContentsAfterCut = scriptContents;
 
       if (context.config.regex) {
-        const regex = new RegExp(context.config.regex);
+        const regex = new RegExp(context.config.regex, 'g');
         const regexResult = regex.exec(scriptContents);
         if (regexResult && regexResult.length > 1) {
           [, scriptContents] = regexResult;
```

Comment:
Also, in this line we only take the first occurrence. Maybe this should also change to fit your needs? We could join all the occurrences and append a `\n` in between, wdyt?